ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/HighAvailabilityTestUtil.java

```diff
@@ -392,11 +400,24 @@ public static void makeAdminRequest(TestKsqlRestApp restApp, final String sql) {
     RestIntegrationTestUtil.makeKsqlRequest(restApp, sql, Optional.empty());
   }
 
+  public static void makeAdminRequest(
```

Comment:
I'm curious why we have a `makeAdminRequest` and `makeAdminRequestWithResponse` here? Both of them call the same callee. Could we just have one `makeAdminRequest` that always returns (a potential empty list), and its callers can just ignore the returned value or not?

Reply:
Sorry, just noticed your question.  I don't think there's a reason to have two separate methods.  I agree with you.  I'll do a quick followup PR to fix this.